src/java/picard/analysis/CollectWgsMetricsFromQuerySorted.java

```diff
@@ -32,18 +32,21 @@
     @Option(shortName = StandardOptionDefinitions.OUTPUT_SHORT_NAME, doc = "Output metrics file.")
     public File OUTPUT;
 
-    @Option(shortName = "USABLE_MQ", doc = "Minimum mapping quality for a read to contribute to usable coverage.", overridable = true)
+    @Option(shortName = "USABLE_MQ", doc = "Minimum mapping quality for a read to contribute to usable coverage.", overridable = true, optional = true)
     public int MINIMUM_USABLE_MAPPING_QUALITY = 20;
 
-    @Option(shortName = "USABLE_Q", doc = "Minimum base quality for a base to contribute to usable coverage.", overridable = true)
+    @Option(shortName = "USABLE_Q", doc = "Minimum base quality for a base to contribute to usable coverage.", overridable = true, optional = true)
     public int MINIMUM_USABLE_BASE_QUALITY = 20;
 
-    @Option(shortName = "RAW_MQ", doc = "Minimum mapping quality for a read to contribute to raw coverage.", overridable = true)
+    @Option(shortName = "RAW_MQ", doc = "Minimum mapping quality for a read to contribute to raw coverage.", overridable = true, optional = true)
     public int MINIMUM_RAW_MAPPING_QUALITY = 0;
 
-    @Option(shortName = "RAW_Q", doc = "Minimum base quality for a base to contribute to raw coverage.", overridable = true)
+    @Option(shortName = "RAW_Q", doc = "Minimum base quality for a base to contribute to raw coverage.", overridable = true, optional = true)
     public int MINIMUM_RAW_BASE_QUALITY = 3;
 
+    @Option(doc = "The number of bases in the genome build of the input file to be used for calculating mean coverage. If not provided, we will assume that ALL bases in the genome should be used (including e.g. Ns)", overridable = true, optional = true)
```

Comment:
i'd change "mean coverage" to "MEAN_COVERAGE" to make it explicit that it is that metric, and only that metric, taht will be influenced by this option